tests-extension/test/qe/README.md

```diff
@@ -192,31 +192,31 @@ All migrated test case code needs the following changes to run in the new test f
 
 ### B. Label Requirements for Migrated and New Cases
 
-#### Required Labels
+#### Required Labels in case title
 1. **Component annotation**: Add `[sig-operator]` in case title
 2. **Jira Component**: Add `[Jira:OLM]` in case title
 3. **OpenShift CI compatibility**: If you believe the case meets OpenShift CI requirements, add `ReleaseGate` label to Ginkgo
    - **Note**: Don't add `ReleaseGate` if case title contains `Disruptive` or `Slow`, or labels contain `StressTest`
 4. **Required For Migrated case from test-private**: Add `[OTP]` in case title
 
-#### Optional Label for Migration and New
-1. **LEVEL0**: Use title label `[Level0]`
-2. **Author**: Deprecated
+#### Optional Labels in Migration/New test cases' title
+1. **LEVEL0**: Add `[Level0]` in the case title, not in the `g.Label()`.
+2. **Author**: Deprecated, remove it.
 3. **ConnectedOnly**: Add `[Skipped:Disconnected]` in title
 4. **DisconnectedOnly**: Add `[Skipped:Connected][Skipped:Proxy]` in title
-5. **Case ID**: change to `PolarionID:xxxxxx`
-6. **Importance**: Deprecated
-7.  **NonPrerelease**: Deprecated
-    - **Longduration**: Change to `[Slow]` in case title
-    - **ChkUpg**: Not supported (openshift-tests upgrade differs from OpenShift QE)
-8.  **VMonly**: Deprecated
-9.  **Slow, Serial, Disruptive**: Preserved
-10. **DEPRECATED**: Deprecated, corresponding cases deprecated. Use `IgnoreObsoleteTests` for deprecation after addition
```

Comment:
@jianzhangbjz it means the old case with DEPRECATED will be deprecated and not migrated.

for already migrated case, if we want to deprecate it late, we add the case full title to IgnoreObsoleteTests in main.go, not add "IgnoreObsoleteTests" in the case title.